tests/Unit/System/Services/ReportServiceTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace System\Services;
+
+use App\System\Report\DTO\ReportDTO;
+use App\System\Report\Repositories\ReportRepositoryInterface;
+use App\System\Report\Services\ReportService;
+use Tests\TestCase;
+
+class ReportServiceTest extends TestCase
+{
+
+    public function testGetReport()
+    {
+        $contentType = 'text';
+
+        $airport = 'fakeAirport';
+
+        $reportDTO = new ReportDTO('fakeAirport', 'text');
+
+        $weatherReportArray = [
+            'title' => 'some data',
+        ];
+
+        $reportRepositoryMock = $this->getMockBuilder(ReportRepositoryInterface::class)
```

Comment:
$reportRepositoryMock = $this->createMock(ReportRepositoryInterface::class);

Reply:
Also, as you have interface for the repository, you can make fake implementation of it for this test